crates/bevy_pbr/src/deferred/mod.rs

```diff
@@ -360,13 +353,13 @@ impl SpecializedRenderPipeline for DeferredLightingLayout {
                 self.bind_group_layout_1.clone(),
             ],
             vertex: VertexState {
-                shader: DEFERRED_LIGHTING_SHADER_HANDLE,
+                shader: self.deferred_lighting_shader.clone_weak(),
```

Comment:
Should we just `clone` instead of `clone_weak`? I understand that technically `clone_weak` is slightly more efficient, but for readability, just doing a `clone` is much less surprising (as in, other people might ask "does clone_weak make sense? Will my shader live long enough here?"). And seeing as how this is probably gonna be the pattern we will copy-paste across the rendering code, it's worth thinking if this `clone_weak` is worth the added confusion.

Reply:
probably negligible, changed